payments-workflows/build.gradle.kts

```diff
@@ -16,9 +21,23 @@ dependencies {
 
 cordapp {
     contract {
-        name("Payment Workflows")
+        name("Payments Workflows")
         vendor("Exactpro Systems LLC")
         licence("Apache License, Version 2.0")
         versionId(1)
     }
 }
+
+tasks.withType<Javadoc> {
```

Comment:
Is it still needed? I think I disabled javadoc tasks in kotlin conventions plugin